cmd/inspect.go

```diff
@@ -92,6 +92,25 @@ func (cli *CLI) inspect(opts Options, args []string) int {
 	cli.formatter.Print(issues, nil, cli.sources)
 
 	if len(issues) > 0 && !force {
```

Comment:
If you cut out the logic into a function, you can write it as follows and I think it will be easier to understand.

```suggestion
	if len(issues) > 0 && !force && exceedsMinimumSeverity(issues, opts.MinimumSeverity) {
```

Reply:
Makes sense, I did this